src/passes/PrintFunctionMap.cpp

```diff
@@ -33,9 +34,14 @@ struct PrintFunctionMap : public Pass {
   bool modifiesBinaryenIR() override { return false; }
 
   void run(PassRunner* runner, Module* module) override {
+    // If an argument is provided, write to that file; otherwise write to
+    // stdout.
+    auto outFile = runner->options.getArgumentOrDefault("symbolmap", "");
```

Comment:
Is this default argument empty string means `cout`? [This](https://github.com/WebAssembly/binaryen/blob/f59174c1cf526d7e9fe5d4ba74ab0370f40a1856/src/support/file.cpp#L91-L92) says `-` is `cout`, so...

Reply:
Interesting, I guess both work then. [Here](https://github.com/WebAssembly/binaryen/blob/f59174c1cf526d7e9fe5d4ba74ab0370f40a1856/src/support/file.h#L53) is a comment documenting the empty file... and the code indeed supports both. We should unify the comments I guess.